app/models/article.rb

```diff
@@ -7,13 +7,20 @@ class Article < ApplicationRecord
   has_many :tags, through: :taggings
   has_many :categorizations, dependent: :destroy
   has_many :categories, through: :categorizations
+  has_many :collection_posts, foreign_key: :collection_id, class_name: :Article
+  belongs_to :collection, foreign_key: :collection_id, class_name: :Article
 
   scope :draft,       -> { where(status: Status.find_by(name: "draft")) }
   scope :edited,      -> { where(status: Status.find_by(name: "edited")) }
   scope :designed,    -> { where(status: Status.find_by(name: "designed")) }
   scope :published,   -> { where(status: Status.find_by(name: "published")) }
 
   scope :live,        -> { where("published_at < ?", Time.now) }
+  scope :recent,      -> { where("published_at BETWEEN ? AND ?", Time.now - 2.days, Time.now) }
+
+  scope :chronological, -> { order(published_at: :desc) }
```

Comment:
Master has a `default_scope` for this.

https://github.com/crimethinc/website/blob/master/app/models/article.rb#L26